### PR TITLE
add new profile for gget

### PR DESCRIPTION
**Pull request by glitsj16**
See https://gget.io/ for details of the application.

### CODE REVIEW

File: etc/profile-a-l/gget.profile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Firejail profile for gget
+# Description:  a cli. to get things. from git repos
```
Comment (rusty-snake) [inline etc/profile-a-l/gget.profile:2]:
```suggestion
# Description: a cli. to get things. from git repos
```

**Review by rusty-snake — COMMENTED** [review]
looks good

File: etc/profile-a-l/gget.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-a-l/gget.profile:2]: Sorry, I just pushed the button when I saw this. Fixed in https://github.com/netblue30/firejail/commit/8a7b96974fd8ddae8ea4641d8e010b1e03b9e817. Thanks!

**Comment by rusty-snake** [comment]
FTR: The missing firecfg.config is intention I guessed.

**Comment by glitsj16** [comment]
> FTR: The missing firecfg.config is intention I guessed.

Indeed. I just stumbled onto this little app when looking for an easy/fast way to get single files from repositories. It isn't even in the AUR, nor packaged by any of the major distros we support. So I decided not to add it to firecfg. Doesn't come with a .desktop file neither. I'll try it out for a while and perhaps add it to the AUR myself. When that happens (or someone else does) I'll revisit this to add firecfg support.

**Pull request merged**